### PR TITLE
Force a crash on startup

### PR DESCRIPTION
**Pull request by stegru**
:construction: :construction: 

Making GPII always crash on start-up.

### CODE REVIEW

**Comment by colinbdclark** [comment]
Wow, users have been clamoring for this feature for months!

**Comment by stegru** [comment]
It's been a hidden feature for some time.

File: index.js [hunk 1]
```diff
@@ -26,6 +26,15 @@ fluid.contextAware.makeChecks({
 });
 
 require("./gpii/node_modules/WindowsUtilities/WindowsUtilities.js");
+
+// Not for production:
```
Comment (JavierJF) [inline index.js]: Love this

**Comment by stegru** [comment]
ok to test

**Comment by stegru** [comment]
Got this twice:
```
[windows-tests] $ /bin/sh -xe /tmp/jenkins7614893886395970249.sh
+ vagrant winrm -c 'do.ps1 -c '\''grunt lint'\'''
Usage: vagrant [options] <command> [<args>]

    -v, --version                    Print the version and exit.
    -h, --help                       Print this help.
```

It's as though winrm plugin isn't in - but it's mentioned in the VM init:

```
    default: WinRM address: 127.0.0.1:5985
```

**Comment by JavierJF** [comment]
> default: Running: provisioning/Installer.ps1 as c:\tmp\vagrant-shell.ps1

Output stopped there, looks like a failure in construction, I'm rechecking the changes. I think I may introduced and error into Installer.ps1 due to a folder name.

**Comment by stegru** [comment]
that's expected: installer.ps1 is now empty

**Comment by gtirloni** [comment]
ok to test

**Comment by stegru** [comment]
Crashed and failed, as expected!

```
Crashing


Stderr from the command:

doitclient.exe : doit: remote error: GetExitCodeProcess failed
    + CategoryInfo          : NotSpecified: (doit: remote er...eProcess failed:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
Build step 'Execute shell' marked build as failure
Sending e-mails to: ci@lists.gpii.net
Finished: FAILURE
```

**Comment by stegru** [comment]
failed with an unrelated error; ok to test

```
    default: /vagrant => /home/jenkins-gpii/workspace/windows-tests
Vagrant can't use the requested machine because it is locked! This
means that another Vagrant process is currently reading or modifying
the machine. Please wait for that Vagrant process to end and try
again. Details about the machine are shown below:
```

**Comment by stegru** [comment]
I made the tests return 0 or 1 via process.exit, and CI still detects it correctly.

The changes to doitclient are a success.

**Comment by amatas** [comment]
ok to test

**Comment by stegru** [comment]
Yes, that's a "successful fail".

The next one will crash the tests, which should make CI fail...

**Comment by stegru** [comment]
It works!

CI now fails when the tests crash

```
Crashing


Stderr from the command:

doitclient.exe : doit: remote error: GetExitCodeProcess failed
    + CategoryInfo          : NotSpecified: (doit: remote er...eProcess failed:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
Build step 'Execute shell' marked build as failure
Sending e-mails to: ci@lists.gpii.net
Finished: FAILURE
```

**Comment by stegru** [comment]
This PR is no longer required

**Pull request closed without merging**